docs/product-user-guide/src/main/asciidoc/rules-attributes-ref.adoc

```diff
@@ -0,0 +1,60 @@
+//Module included in the following assemblies:
+//product-assembly_guided-rules/../main.adoc
+//product-assembly_guided-rule-templates/../main.adoc
+//product-user-guide/../chap-writing-rules
+
+[id='rules-attributes-ref_{context}']
+= Rule attributes
+
+The following table lists the names and supported values of the attributes that you can assign to business rules:
+
+*<@Michael (3 of 6): Could you again validate each of these attribute descriptions? It's like the one from the uploaded decision tables doc, but I've revised it pretty substantially for more clarity. I'll make the two tables more consistent once I get your go-ahead. I've also put in a UX request to just put in some help text in the UI for this, and they'll use what you/I/we come up with here. I have a few specific questions along the way that you'll see, but I've revised basically all of them, just as a heads-up.>*
+
+.Rule attributes
+[cols="30%,70%", options="header"]
+|===
+|Attribute
+|Value
+
+|`salience`
+|An integer defining the priority of the rule. Rules with a higher salience value are given higher priority when ordered in the activation queue.
+
+|`enabled`
+|A Boolean value. When the option is selected, the rule is enabled. When the option is not selected, the rule is disabled.
+
+|`date-effective`
+|A string containing a date and time definition. The rule can be activated only if the current date and time is after a `date-effective` attribute.
+
+|`date-expires`
+|A string containing a date and time definition. The rule cannot be activated if the current date and time is after the `date-expires` attribute.
+
+|`no-loop`
+|A Boolean value. When the option is selected, the rule cannot be reactivated (looped) if a consequence of the rule re-triggers a previously met condition. When the condition is not selected, the rule can be looped in these circumstances.
+
+|`agenda-group`
+|A string identifying an agenda group to which you want to assign the rule. Agenda groups allow you to partition the agenda to provide more execution control over groups of rules. Only rules in the agenda group that have acquired a focus are able to be activated. *<@Michael (4 of 6): Can you tell me what a focus is and what it's used for? It comes up again in `auto-focus`.>*
```

Comment:
"Only rules in the agenda group that **has** acquired focus..."

Reply:
Agendas et al are a complex topic. It might be worth just adding a link to other sections of the User Guide. See https://docs.jboss.org/drools/release/7.5.0.Final/drools-docs/html_single/index.html#_execution_control (Community Documentation) specifically "6.2.1. Agenda" and "6.2.2.4. Agenda Groups"

Reply:
After these docs are done, I plan to go back separately and add an Advanced chapter of sorts where applicable and include content from the Drools doc, to dive in more deeply to things like this as needed.